slave.mk

```diff
@@ -376,6 +376,10 @@ $(info "USE_NATIVE_DOCKERD_FOR_BUILD"    : "$(SONIC_CONFIG_USE_NATIVE_DOCKERD_FO
 $(info "SONIC_USE_DOCKER_BUILDKIT"       : "$(SONIC_USE_DOCKER_BUILDKIT)")
 $(info "USERNAME"                        : "$(USERNAME)")
 $(info "PASSWORD"                        : "$(PASSWORD)")
+$(info "SECURE_UPGRADE_MODE"             : "$(SECURE_UPGRADE_MODE)")
+$(info "SECURE_UPGRADE_DEV_SIGNING_KEY"  : "$(SECURE_UPGRADE_DEV_SIGNING_KEY)")
+$(info "SECURE_UPGRADE_DEV_SIGNING_CERT" : "$(SECURE_UPGRADE_DEV_SIGNING_CERT)")
+$(info "SECURE_UPGRADE_PROD_SIGNING_TOOL": "$(SECURE_UPGRADE_PROD_SIGNING_TOOL)")
```

Comment:
Update on HLD is needed for SECURE_UPGRADE_PROD_SIGNING_TOOL

Reply:
And also PR description should be updated for SECURE_UPGRADE_PROD_SIGNING_TOOL

Reply:
SECURE_UPGRADE_DEV_SIGNING_KEY is a secure-boot key, Kernel module signing may use the same key or a different key. If the build system exposes SECURE_UPGRADE_KO_SIGNING_KEY then the vendor can have the flexibility to choose a separate key or pass both variables with the same keys.

Reply:
In our HLD we are using the same private key for signing boot loaders components and kernel modules, this is in order to make it more easily from the user's POV.
If you propose to enhance it and have multiple keys, I suggest doing a future meeting about it.
For now, in this PR, let's save the HLD conventions about it.

Reply:
Okay. Ack